configs/common/Simulation.py

```diff
@@ -733,4 +733,4 @@ def run(options, root, testsys, cpu_class):
         m5.checkpoint(joinpath(cptdir, "cpt.%d"))
 
     if exit_event.getCode() != 0:
-        print("Simulated exit code not 0! Exit code is", exit_event.getCode())
+        fatal("Simulated exit code is %s" % exit_event.getCode())
```

Comment:
This looks like an unrelated change.

Reply:
Yes I was using this to detect when a test failed by checking the process return code. I've dropped this commit.